### PR TITLE
Feature/upgrade steal

### PR DESCRIPTION
**Pull request by matthewp**

### CODE REVIEW

**Comment by matthewp** [comment]
One test is failing in Firefox with Testee. Others are passing. Still looking into it.

**Comment by matthewp** [comment]
@Bajix I don't think this change needs a release, right?

**Comment by matthewp** [comment]
Oh, nm, it does. I'll do a prelease.

File: test/basics/test.html [hunk 1]
```diff
@@ -13,9 +15,15 @@
 		var canViewModel = require("can-view-model");
 		var Zone = require("can-zone");
 
-		new Zone().run(function() {
-			return System.import('test/basics/index.stache!done-autorender');
-		}).then(function() {
+		System.import('test/basics/index.stache!done-autorender')
```
Comment (Bajix) [inline test/basics/test.html]: See: https://github.com/canjs/can-zone/issues/98

**Pull request merged**